dev-libs/sway/sway-9999.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2016 Gentoo Foundation
```

Comment:
I'd request that you just dump this ebuild unless it's really needed.  Live ebuilds tend to cause unnecessary headaches in the support channels.